app/src/main/java/org/opencord/aaa/impl/AaaManager.java

```diff
@@ -296,7 +338,14 @@ public void handleRadiusPacket(RADIUS radiusPacket)
 
         EAP eapPayload;
         Ethernet eth;
-
+        boolean isValid = isValidValidator(radiusPacket);
+        if (!isValid) {
```

Comment:
To be consistent, if you are not using isValid flag anywhere else in this method, why not rename the isValidValidator method to checkReceivedPacketForValidValidator() and increment the counter in that method itself.